frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -28,6 +29,10 @@ export const FilterTypes = {
   keyword: 'keyword',
 };
 
+// Mapping filter retention preference per filterKey
+export const mapFilterRetention = {};
+mapFilterRetention[devKey] = ['kind']; // only retain kind-type filter preference for Dev Catalog
```

Comment:
The `devKey` is specific to dev-catalog and shouldn't be referenced in the base class.

Pass in the retention array as props.

Reply:
now passing as a prop